### DISCUSSION TITLE
如何进入 Feedly / Inoreader 才能看到简悦的注入 icon

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#1507] [使用技巧]
> 关于此功能的详细说明 [请看这里](http://ksria.com/simpread/docs/#/RSSReader)

## 原因

因为 Feedly 或 Inoreader 有很多入口方式，**简悦使用了 Dom 变化跟踪的的方式进行注入**，所以一旦出现 **某些未判断的操作步骤** 就会无法工作。

## 解决方法（Feedly）

> 已知可用的操作步骤

Setting → General → Slider

![image](https://user-images.githubusercontent.com/81074/101716561-7aa65480-3ad8-11eb-89e6-e3408839aecb.png)

![image](https://user-images.githubusercontent.com/32646062/101712019-a2dd8580-3acf-11eb-830a-24162ecbd78a.png)  

**进入 Feedly 后仅支持通过通过侧栏 FEEDS 进入**，如下图所示：

<img width="1552" alt="Xnip2020-12-09_14-30-20" src="https://user-images.githubusercontent.com/81074/101593807-88ea6700-3a2b-11eb-9e9f-811405715247.png">

PS：**如果按照步骤操作仍无法工作，请刷新页面后再次进入**。

## 解决方法（Inoreader）

> 设置文章为列表形式

![image](https://user-images.githubusercontent.com/81074/130751991-7a3e39a4-55b1-47bd-a6da-49501e07b7ac.png)

> 已知好使的操作步骤 → **进入 Inoreader 后仅支持通过通过侧栏 订阅源 方式进入**，如下图所示：

<img width="1564" alt="Xnip2020-12-09_14-39-50" src="https://user-images.githubusercontent.com/81074/101594339-7886bc00-3a2c-11eb-8bec-378990a85ee1.png">

## 后续

下个版本会加入更多的进入 Feedly / Inoreader 方案。

## 关联

- [在 Feedly / Inoreader 中如何方便的使用「✂ 简悦的一键收藏」 ](https://github.com/Kenshin/simpread/discussions/1559)

**Comment by psterman** [comment]
chrome88 win10 简悦2.0 
还是没看到那个按钮

**Reply by Kenshin** [reply]
请升级到简悦 2.1 版本，功能详细说明请看 http://ksria.com/simpread/docs/#/RSSReader

另外，此功能不是很稳定，原因就是上面的描述中。

**Reply by zero617** [reply]
简悦2.2 inoreader 看不见icon

**Reply by Kenshin** [reply]
> 简悦2.2 inoreader 看不见icon

估计是 inoreader 的结构改变造成的，这个功能非常依赖页面结构。
